src/tox_lsr/test_scripts/runansible-test.sh

```diff
@@ -64,6 +64,11 @@ for file in "$TOXINIDIR"/.sanity-ansible-ignore-*.txt; do
   fi
 done
 
+if [ "${LSR_ANSIBLE_TEST_DOCKER:-false}" = true ]; then
+  ansible-test sanity --docker 2>&1 | ansible_test_filter
+  exit 0
```

Comment:
This means if LSR_ANSIBLE_TEST_DOCKER is true, there is no way to check the result of ansible-test sanity test?

Reply:
> This means if LSR_ANSIBLE_TEST_DOCKER is true, there is no way to check the result of ansible-test sanity test?

not sure what you mean - if any of the sanity tests give an error, then `ansible-test` will exit with a non-zero exit code, and due to the use of `set -e`, the script will exit with an error and tox will report the error

Reply:
well `set -e` and `-o pipefail`

Reply:
:p  You are right.